HotFix/Core/Session.cs

```diff
@@ -28,6 +29,7 @@ public Session(IConfiguration configuration, IClock clock, ITransport transport,
                 InboundTimestamp = clock.Time,
                 OutboundTimestamp = clock.Time
             };
+            Active = true;
```

Comment:
I don't think it makes sense for this to be set true by default - it should probably be set to true at the end of the `logon procedure`, aka once we know we're logged on...